src/java/voldemort/server/protocol/vold/ClientRequestHandler.java

```diff
@@ -0,0 +1,138 @@
+package voldemort.server.protocol.vold;
```

Comment:
Can we change the package name to 
"package voldemort.server.protocol.native"  instead of "
"package voldemort.server.protocol.vold" ?